app/src/main/java/com/onboarding/estacionAR/activities/MainActivity.java

```diff
@@ -1,14 +1,16 @@
-package com.onboarding.parkingsystemjava;
+package com.onboarding.estacionAR.activities;
```

Comment:
the packages will be always in the singular. The correct is **activity** not **activities**

Reply:
Acknowledged and chenged.